### DISCUSSION TITLE
[6 luglio 2023] Automi

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/automi-calcolabilita-complessita#13] [Automi]
![image](https://github.com/sapienzastudentsnetwork/acc2324/assets/69389461/9332d288-1c75-40fa-944f-375c8419cbd0)

**Comment by Exyss** [comment]
Per il primo punto, ripropongo la soluzione presente sui miei appunti:

![image](https://github.com/sapienzastudentsnetwork/acc2324/assets/65043207/9145e20a-5da4-401b-b78d-dc0a4c62e94d)


----

La dimostrazione richiesta nel secondo punto è presente sul libro di testo

**Reply by daromacarlo** [reply]
<img width="1107" height="657" alt="immagine" src="https://github.com/user-attachments/assets/e847f1c0-0c8d-4c53-b0c7-50c232a53195" />
